### DISCUSSION TITLE
Difference between this and AspNetIdentity?

### DISCUSSION THREAD

**Discussion by kristofferth** [AzureAD/microsoft-identity-web#1703] [Q&A] [answered]
I don't fully understand the difference between this microsoft-identity-web and AspNetIdentity as seen on https://github.com/aspnet/AspNetIdentity

AspNetIdentity seems a lot older. Which framework should be used if you just wanted to make an application where users could login and see a dashboard of data personalized to them?

Thanks

**Comment by kristofferth** [comment]
bump

**Comment by jmprieur** [comment]
@kristofferth [kristofferth](https://github.com/kristofferth)
What is AspNetIdentity?
Do you have some docs?
Microsoft.Identity.Web is the long term supported library to sign-in users, protect web APIs and call downstream web APIs with Azure AD.
See https://github.com/AzureAD/microsoft-identity-web/wiki/1.0.0

**Reply by kristofferth** [reply]
Here is the docs on AspNetIdentity https://docs.microsoft.com/en-us/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity

**Comment by jmprieur** [comment, marked as answer]
@kristofferth : this was for ASP.NET (Framework)
For the moment, Microsoft.Identity.Web is for ASP.NET Core
